patterns/2-structured/fixing-issues-step-by-step.md

```diff
@@ -0,0 +1,43 @@
+## Title
+
+Fixing Issues Step by Step
+
+## Patlet
+
+Solving an issue by deadline without consideration of the current scope of change to process causes developer frustration and decreased efficiencies. Utilize a continuous improvement approach which right-sizes the work and provides realistic expectations.
+
+## Problem
+
+Results are expected to take place within a few days/weeks after a process has started. Adding a process or a toolchain by itself will not provide any result by itself beyond developer frustration. Stating that something has to work by a certain deadline will not make it possible, this will just force the situation and workarounds will be done by developers.
+
+## Forces
+
+Understanding step changes in a process requires adequate time and clarity of impact
+Clarity of impact to the process and delivery pressure are deterrents to change
+
+## Solutions
+
+When planning new steps, set the way that step is successful if compared to others. Follow a continuous improvement approach and measure the delta between steps.
+
```

Comment:
When reviewing contributions, don't strive for perfection and approve them as soon as there's a tangible improvement (regressions must be avoided, of course).

Lurkers will feel that the (small) achievement is within their reach and will convince them to contribute. This bootstraps a virtuous cycle, as more, more frequent, and more diverse contributions also seduce other lurkers to join the party.